Makefile fails when building `/example/standalone`

## Current behaviour
After PR #42  the code fails when running `make dev-standalone` or `make test-standalone` or `make up-standalone`. The error is:
```bash
TASK [Put Secrets with Vault] **************************************************
fatal: [default]: FAILED! => {}

MSG:

The task includes an option with an undefined variable. The error was: 'user' is undefined

The error appears to be in '/vagrant/dev/ansible/01_put_secrets_vault.yml': line 2, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- name: Put Secrets with Vault
  ^ here


NO MORE HOSTS LEFT *************************************************************

PLAY RECAP *********************************************************************
default                    : ok=4    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0   

Ansible failed to complete successfully. Any error output should be
visible above. Please fix these errors and try again.
make: *** [up-standalone] Error 1
```

This happened because Ansible playbook runs (`terraform-nomad-nifiregistry/dev/ansible/*`, and expects that the parameters `repo` , `branch` , `user` and `token` have a value. The parameters have only value when running example `example/standalone_git`. 

## How to reproduce?
Run one of these commands: `make dev-standalone` or `make test-standalone`or `make up-standalone`

## Suggestion(s)/solution(s) [Optional]
The Ansible playbooks (file: `01_put_secrets_vault.yml` and `02_put_variables_consul.yml`) should not be executed when running the example/standalone

## Checklist (after created issue)
- [ ] Added label(s)
- [ ] Added to project
- [ ] Added to milestone

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -9,7 +9,7 @@
 
 ### Updated 
 - Updated tests with parameters #39
-- Changed Ansible playbooks `01_put_secrets_vault.yml` and `02_put_variables_consul.yml`, such that they only execute for example/standalone_git. 
+- Changed Ansible playbooks `01_put_secrets_vault.yml` and `02_put_variables_consul.yml`, such that they only execute for example/standalone_git. #43
 
 ## [0.2.1]
 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -9,6 +9,7 @@
 
 ### Updated 
 - Updated tests with parameters #39
+- Changed Ansible playbooks `01_put_secrets_vault.yml` and `02_put_variables_consul.yml`, such that they only execute for example/standalone_git. 
 
 ## [0.2.1]
 

--- a/dev/ansible/01_put_secrets_vault.yml
+++ b/dev/ansible/01_put_secrets_vault.yml
@@ -1,3 +1,6 @@
+- name: Running example/{{ mode }}
+  set_fact:
+    mode: "{{ mode }}"
 
 - name: Put Secrets with Vault
   shell: vault kv put secret/github git_access_user="{{ user }}" git_access_password="{{ token }}"
@@ -5,3 +8,4 @@
   environment:
     VAULT_ADDR: "{{ lookup('env', 'VAULT_ADDR') }}"
     VAULT_TOKEN: "{{ lookup('env', 'vault_master_token') }}"
+  when: mode == "standalone_git"

--- a/dev/ansible/02_put_variables_consul.yml
+++ b/dev/ansible/02_put_variables_consul.yml
@@ -1,12 +1,19 @@
+- name: Running example/{{ mode }}
+  set_fact:
+    mode: "{{ mode }}"
+
 - name: Put repo value in Consul Key Value Store
   shell: consul kv put github/repo "{{ repo }}"
   run_once: true
   environment:
     CONSUL_HTTP_ADDR: "{{ lookup('env', 'CONSUL_HTTP_ADDR') }}"
     CONSUL_HTTP_TOKEN: "{{ lookup('env', 'consul_master_token') }}"
+  when: mode == "standalone_git"
+
 - name: Put branch value in Consul Key Value Store
   shell: consul kv put github/branch "{{ branch }}"
   run_once: true
   environment:
     CONSUL_HTTP_ADDR: "{{ lookup('env', 'CONSUL_HTTP_ADDR') }}"
     CONSUL_HTTP_TOKEN: "{{ lookup('env', 'consul_master_token') }}"
+  when: mode == "standalone_git"

